src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java

```diff
@@ -35,11 +59,14 @@ public ZipFileProcessingResult process(
             switch (FilenameUtils.getExtension(zipEntry.getName())) {
                 case "json":
                     metadata = toByteArray(extractedZis);
-
+                    log.info(
+                        "File: {}, Meta data size: {}",
+                        zipFileName,
+                        FileUtils.byteCountToDisplaySize(metadata.length)
+                    );
                     break;
                 case "pdf":
-                    pdfs.add(new Pdf(zipEntry.getName(), toByteArray(extractedZis)));
-
+                    pdfs.add(new Pdf(zipEntry.getName(), null));
```

Comment:
Names and sizes of pdfs should also be logged - here or after the while loop.

Reply:
we do not have the size here, I will refactor this processor soon.

Reply:
at least the names should be logged.

Reply:
name and size added to related method.